packages/driver/src/multi-domain/index.ts

```diff
@@ -29,13 +29,13 @@ const onCommandEnd = (command: Cypress.CommandQueue) => {
 }
 
 const onLogAdded = (attrs) => {
-  specBridgeCommunicator.toPrimary('command:update', {
+  specBridgeCommunicator.toPrimary('log:added', {
     logAdded: $Log.toSerializedJSON(attrs),
   })
 }
 
 const onLogChanged = (attrs) => {
-  specBridgeCommunicator.toPrimary('command:update', {
+  specBridgeCommunicator.toPrimary('log:changed', {
     logChanged: $Log.toSerializedJSON(attrs),
   })
```

Comment:
Since these are discrete events now, let's just pass along the serialized log attrs directly and skip the `logAdded` and `logChanged` properties. e.g:

```js
  specBridgeCommunicator.toPrimary('log:added', $Log.toSerializedJSON(attrs))
```

Reply:
Good thinking!